pkg/cmd/scan_ciskubebench.go

```diff
@@ -69,11 +69,20 @@ func ScanKubeBenchReports(cf *genericclioptions.ConfigFlags) func(cmd *cobra.Com
 		}
 
 		writer := kubebench.NewReadWriter(kubeClient)
+		argsMap := make(map[string]string)
+		for _, s := range args {
+			argsMap[s] = s
+		}
 
 		// TODO Move this logic to scanner.ScanAll() method. We should not mix discovery / scanning logic with the CLI.
 		var wg sync.WaitGroup
 
 		for _, node := range nodeList.Items {
+			_, found := argsMap[node.Name]
+			if !found && len(args) > 0 {
+				klog.V(3).Infof("Node not found. Skipping the scan ciskubenchreports")
```

Comment:
If the specified node does not exist this error will be duplicated x number of nodes in the cluster. For example, on 3 nodes cluster you'll see:

```console
$ starboard scan ciskubebenchreports kind-workerr -v 3
I0504 14:22:15.008979   13121 scan_ciskubebench.go:83] Node not found. Skipping the scan ciskubenchreports
I0504 14:22:15.009172   13121 scan_ciskubebench.go:83] Node not found. Skipping the scan ciskubenchreports
I0504 14:22:15.009176   13121 scan_ciskubebench.go:83] Node not found. Skipping the scan ciskubenchreports
```

I think we should list all nodes only if the `len(args)` equals zero. Otherwise we should iterate through args and get nodes one by one.

Reply:
I added another control to skip the klog.V(3).Infof